### PR TITLE
feature(files): Hide breadcrumbs when an upload is ongoing on narrow screen #40942

### LINKED ISSUE #40942: [F2V] Hide breadcrumbs when an upload is ongoing on narrow screen
image

### PR DESCRIPTION
**Pull request by arublov**
* Resolves: #40942 

Before:
Screenshot 2024-02-04 at 02 22 48

After:
Screenshot 2024-02-04 at 02 14 17

### CODE REVIEW

**Review by nfebe — APPROVED** [review]

File: apps/files/src/components/BreadCrumbs.vue [hunk 1]
```diff
@@ -108,6 +118,15 @@ export default defineComponent({
...
 			})
 		},
+
+		uploadInProgress(): boolean {
+			return this.uploaderStore.queue.length !== 0
+		},
+
+		shouldShowBreadcrumbs(): boolean {
+        	return this.filesListWidth < 768 && !this.uploadInProgress
+    	},
```
Comment (skjnldsv) [inline apps/files/src/components/BreadCrumbs.vue:128]:
```suggestion
			return this.filesListWidth < 768 && !this.uploadInProgress
		},
```

File: apps/files/src/components/BreadCrumbs.vue [hunk 2]
```diff
@@ -108,6 +118,15 @@ export default defineComponent({
...
+
```
Comment (skjnldsv) [inline apps/files/src/components/BreadCrumbs.vue:129]:
```suggestion

		isUploadInProgress(): boolean {
			return this.uploaderStore.queue.length !== 0
		},

		// Hide breadcrumbs if an upload is ongoing on arrow screens
		shouldShowBreadcrumbs(): boolean {
			return this.filesListWidth < 768 && !this.isUploadInProgress
		},
```

Comment (arublov) [reply apps/files/src/components/BreadCrumbs.vue]: Yeah right, I'll check tabs/space next time and stick to your all suggestions. Thank you @skjnldsv !
Any additional action from my side needed on this? If not, I'll go do the next task :)

**Review by skjnldsv — APPROVED** [review]

File: apps/files/src/components/BreadCrumbs.vue (hunk 2 above)
Comment (skjnldsv) [reply apps/files/src/components/BreadCrumbs.vue]: Seems good to me! :rocket:

**Comment by skjnldsv** [comment]
please rebase and compile assets :)

**Review by skjnldsv — APPROVED** [review]

**Comment by arublov** [comment]
@skjnldsv , sorry for the delay, assets have now been pushed and rebased branch.

**Pull request merged**